src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/model/UserModel.java

```diff
@@ -62,16 +62,16 @@ public void setUserEmail(String userEmail) {
         this.userEmail = userEmail;
     }
 
```

Comment:
there shouldn't be commented out lines in da code

Reply:
thx, deleted!